src/test/resources/features/ifs/instalment_calculation/InstalmentCalculationMultipleDebts.feature

```diff
@@ -81,6 +101,29 @@ Feature: Instalment calculation for multiple debts - Input 1 & 2
       | 2                | 2020-08-01 | 6000      |
       | 7                | 2020-12-01 | 2187      |
 
+  @DTD-3163
+  Scenario: interestStartDate is included but not in the Future, then quote date should be used
+    Given debt instalment calculation with 129 details
+      | instalmentPaymentAmount | instalmentPaymentDate | paymentFrequency | interestCallDueTotal | quoteType | quoteDate  |
+      | 6000                    | 2025-06-30            | monthly          | 1000                 | duration  | 2025-05-31 |
+    And the instalment calculation has no postcodes
+    And debt plan details with initial payment
+      | initialPaymentAmount | initialPaymentDate |
+      | 5000                 | 2025-06-01         |
+    And the instalment calculation has debt item charges
+      | debtId     | debtAmount | mainTrans | subTrans | interestStartDate |
+      | TPSSDebt1  | 16000      | 1525      | 1000     | 2025-01-01        |
+      | DRIERDebt1 | 14000      | 1525      | 1000     | 2025-03-01        |
+    When the instalment calculation detail is sent to the ifs service
+    Then the instalment calculation summary contains values
+      | numberOfInstalments | duration | interestAccrued | planInterest | totalInterest |
+      | 7                   | 5        | 1000            | 873          | 1873          |
+    And IFS response contains expected values
```

Comment:
Would be useful to include interestRate here as you have done with the above scenarios?